python/paddle/fluid/tests/unittests/dist_mnist.py

```diff
@@ -66,8 +68,7 @@ def cnn_model(data):
         size=SIZE,
         act="softmax",
         param_attr=fluid.param_attr.ParamAttr(
-            initializer=fluid.initializer.NormalInitializer(
-                loc=0.0, scale=scale, seed=1)))
+            initializer=fluid.initializer.Constant(value=0.1)))
```

Comment:
why change this?

Reply:
pserver side initializer may create different values on CPU/GPU